src/app/calendar/samplePage.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import React from "react";
+import Calendar, { CalendarEvent } from "../../components/Calendar/Calendar";
+import { styled } from "styled-components";
+
+const Centerer = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+const SizedAspectRatio = styled.div`
+    width: 100vmin;
+    height: 100vmin;
+`;
+
+const SampleCalendar: React.FC<{}> = () => {
+    const sampleEvents: CalendarEvent[] = [
+        {
+            id: "a",
+            title: "event1",
+            start: new Date("2023-07-11"), // unless a timezone is specified, the ISO8601 standard assumes UTC (and therefore so does the date constructer). the calendar will then display the date in the local timezone,
+            end: new Date("2023-07-12"), // which is why the date displayed is 1 hour later than the date specified when in BST. When printing to the console, a JS Date object will also be displayed in the local timezone but is internally stored as UTC.
```

Comment:
can these comments be a separate multi-line comment not in the JSON list? this is great context but dislike it being 'attached' to different lines

also, does the `getHours` function return the UTC hour, if it's internally stored as it, or the locale hour? what about the day (if we are within an hour of midnight / in a different timezone)?